src/orion/core/worker/experiment.py

```diff
@@ -257,11 +258,12 @@ def reserve_trial(self, score_handle=None):
             selected_trial = self.reserve_trial(score_handle=score_handle)
         else:
             selected_trial = Trial(**selected_trial_dict)
+            TrialMonitor(self, selected_trial.id).start()
 
         return selected_trial
 
     def push_completed_trial(self, trial):
-        """Inform database about an evaluated `trial` with results.
+        """Inform database about an evaluated `trial` with resultlts.
```

Comment:
Not sure this is an improvement :rofl:

Reply:
I fixed it (here)[https://github.com/Epistimio/orion/pull/203/files#diff-6d901bae3fe0c9e74e137277f6fdf9b4L264] ahaha